### ISSUE TITLE
EDI file with frequencies in increasing order does not re-order the tipper

### ISSUE THREAD

**Issue by JKutt** [MTgeophysics/mtpy-v2#50]
in `edi.py` the code that checks if the frequency is increasing or decreasing only the impedance data is modified. https://github.com/kujaku11/mt_metadata/blob/6936e35b26ad9dbb1bf272bf12bf71b22ac8794a/mt_metadata/transfer_functions/io/edi/edi.py#L459-L465

I would be happy to start a PR to start discussion on contributing and how it would like to be done.

## Expected Behavior
If tipper data is available in the edi file and the frequencies are in increasing order, the loader re-orders to deceasing frequency and that the data for both tipper and impedance are re-ordered.

## Current Behavior
If the EDI file has the frequencies in increasing order, the import flips the frequency vector and impedance tensor entries to decreasing in frequency but it fails to flip the tipper data.

Suggested changes would be to add check if tipper data is available and if so add:

```
self.t = self.t[::-1]
self.t_err = self.t_err[::-1]
```

## Steps to Reproduce (for bugs)

1. Have an EDI file with increasing frequency order.
2. Create an EDI object .
3. Use EDI object to read EDI file with increasing order.
4. Plot the tipper transfer function and compare to data in EDI file.

## Your Environment

  * Operating system: Linux
  * MtPy version: 2.0.6
  * Python version: 3.10.12

**Comment by kujaku11** [comment]
@JKutt This is great, thanks for catching this error.  It should be an easy fix in `mt_metadata` I'll link to your issue in a new PR there.

**Comment by kujaku11** [comment]
@JKutt This issue has been fixed (hopefully) in `mt-metadata`(v0.3.8) available on PyPi and conda (in a day or so).  

I added a specific function to check for descending frequency order in the transfer function when reading in an EDI that now flips the tipper as well.  I added a simple test too.  See https://github.com/kujaku11/mt_metadata/pull/228 for more details.  Once you confirm it works we can close this issue?
